### PR TITLE
l10n: uk: update core translation

### PR DESCRIPTION
**Pull request by kasya**

### CODE REVIEW

File: po/uk.po [hunk 1]
```diff
@@ -1732,22 +1732,22 @@ msgstr ""
...
 #, c-format
 msgid "%s tracks both %s and %s"
-msgstr ""
+msgstr "%s відстежує як %s, так і %s"
 
 #: remote.c:824
 #, c-format
 msgid "key '%s' of pattern had no '*'"
-msgstr ""
+msgstr "ключі '%s' шаблону не містили '*'"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "ключ '%s' шаблону не містив '*'"
```

File: po/uk.po [hunk 2]
```diff
@@ -1732,22 +1732,22 @@ msgstr ""
...
 
 #: remote.c:834
 #, c-format
 msgid "value '%s' of pattern has no '*'"
-msgstr ""
+msgstr "значення '%s' шаблону не має '*'"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "значення '%s' шаблону не містить '*'"
```

File: po/uk.po [hunk 3]
```diff
@@ -1808,16 +1817,19 @@ msgid ""
...
 msgstr ""
+"Частина <src> у refspec є об’єктом blob.\n"
+"Ви хотіли позначити новий blob, надіславши зміни до\n"
+"'%s:refs/tags/%s'?"
 
 #: remote.c:1311
 #, c-format
 msgid "%s cannot be resolved to branch"
-msgstr ""
+msgstr "%s не може бути перетворено на гілку"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "%s неможливо розвʼязати в гілку"
```

File: po/uk.po [hunk 4]
```diff
@@ -1808,16 +1817,19 @@ msgid ""
...
 
 #: remote.c:1322
 #, c-format
 msgid "unable to delete '%s': remote ref does not exist"
-msgstr ""
+msgstr "не вдається видалити '%s': віддалене посилання не існує"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "не вдалося видалити '%s': віддалене посилання не існує"
```

File: po/uk.po [hunk 5]
```diff
@@ -1831,32 +1843,33 @@ msgstr ""
...
 #: remote.c:1862 remote.c:1969
 msgid "HEAD does not point to a branch"
-msgstr ""
+msgstr "HEAD не вказує на гілку"
 
 #: remote.c:1871
 #, c-format
 msgid "no such branch: '%s'"
-msgstr ""
+msgstr "не існує такої гілки: '%s'"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "немає такої гілки: '%s'"
```

File: po/uk.po [hunk 6]
```diff
@@ -1831,32 +1843,33 @@ msgstr ""
...
 #, c-format
 msgid "no upstream configured for branch '%s'"
-msgstr ""
+msgstr "репозиторій рівнем вишче не налаштовано для гілки '%s'"
 
 #: remote.c:1880
 #, c-format
 msgid "upstream branch '%s' not stored as a remote-tracking branch"
-msgstr ""
+msgstr "гілка '%s' репозиторієм рівнем вище не зберігається як гілка з віддаленим відстеженням"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "гілка '%s' репозиторія рівнем вище не збережена як гілка віддаленого відстеження"
```

File: po/uk.po [hunk 7]
```diff
@@ -1831,32 +1843,33 @@ msgstr ""
...
 
 #: remote.c:1895
 #, c-format
 msgid "push destination '%s' on remote '%s' has no local tracking branch"
 msgstr ""
+"шлях призначення для надіслання '%s' на віддаленому '%s' не має локальної гілки відстеження"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
"шлях призначення надсилання '%s' на віддаленому '%s' не має гілки локального відстеження"
```

File: po/uk.po [hunk 8]
```diff
@@ -1831,32 +1843,33 @@ msgstr ""
...
 
 #: remote.c:1910
 #, c-format
 msgid "branch '%s' has no remote for pushing"
-msgstr ""
+msgstr "гілка '%s' не має віддаленого репозиторія для надіслання"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "гілка '%s' не має віддаленого репозиторія для надсилання"
```

File: po/uk.po [hunk 9]
```diff
@@ -1869,12 +1882,12 @@ msgstr ""
 
 #: remote.c:1955
 msgid "cannot resolve 'simple' push to a single destination"
-msgstr ""
+msgstr "неможливо вирішити 'простe' надіслання до одного пункту призначення"
```
Comment (arkid15r) [inline po/uk.po]:
```suggestion
msgstr "неможливо вирішити 'простe' надсилання до одного пункту призначення"
```

**Review by arkid15r — COMMENTED** [review]
LGTM

File: po/uk.po (hunk 6 above)
Comment (kasya) [reply po/uk.po]: 🤯

**Pull request merged**